src/org/parosproxy/paros/extension/ExtensionHookMenu.java

```diff
@@ -120,6 +121,16 @@
      */
     private List<JMenuItem> toolsMenuItemList;
 
+    /**
+     * The import menus added to this extension hook.
+     * <p>
+     * Lazily initialised.
+     *
+     * @see #addImportMenuItemImpl(JMenuItem)
+     * @see #getImport()

```

Comment:
A `@since TODO Add version` would be handy. (IMHO)

(Left as TODO since we haven't 100% decided what the next release is, there's always a release task to do a search/replace)